stlTranslator/HeaderFiles/triangulation.h

```diff
@@ -14,4 +14,4 @@ class Triangulation {
 
     void processString(const std::string& data);
     std::vector<Triangle> createTriangles(const std::string& data); 
-};
\ No newline at end of file
+};
```

Comment:
Triangulation class should have the functionality only to triangulate the points and triangles, not store data.